Modules/System/Azure Blob Services API/src/ABSBlobClient.Codeunit.al

```diff
@@ -146,10 +161,24 @@ codeunit 9053 "ABS Blob Client"
     /// <param name="BlobName">The name of the blob.</param>
     /// <param name="SourceInStream">The Content of the Blob as InStream.</param>
     /// <param name="ABSOptionalParameters">Optional parameters to pass.</param>
-    /// <returns>An operation reponse object</returns>
+    /// <returns>An operation response object</returns>
     procedure PutBlobBlockBlobStream(BlobName: Text; var SourceInStream: InStream; ABSOptionalParameters: Codeunit "ABS Optional Parameters"): Codeunit "ABS Operation Response"
     begin
-        exit(ABSClientImpl.PutBlobBlockBlobStream(BlobName, SourceInStream, ABSOptionalParameters));
+        exit(ABSClientImpl.PutBlobBlockBlobStream(BlobName, SourceInStream, '', ABSOptionalParameters));
+    end;
+
+    /// <summary>
+    /// Uploads the content of an InStream as a BlockBlob.
+    /// see: https://go.microsoft.com/fwlink/?linkid=2210387
+    /// </summary>
+    /// <param name="BlobName">The name of the blob.</param>
+    /// <param name="SourceInStream">The Content of the Blob as InStream.</param>
+    /// <param name="ContentType">Value for Content-Type HttpHeader (e.g. 'text/plain; charset=UTF-8')</param>
+    /// <param name="ABSOptionalParameters">Optional parameters to pass.</param>
+    /// <returns>An operation response object</returns>
+    procedure PutBlobBlockBlobStream(BlobName: Text; var SourceInStream: InStream; ContentType: Text; ABSOptionalParameters: Codeunit "ABS Optional Parameters"): Codeunit "ABS Operation Response"
+    begin
+        exit(ABSClientImpl.PutBlobBlockBlobStream(BlobName, SourceInStream, '', ABSOptionalParameters));
```

Comment:
I think the ContentType parameter should be used here?

Reply:
@pri-kise - yes, sure. Sorry, missed that one. Updated.